src/Schema/Directives/Nodes/UnionDirective.php

```diff
@@ -46,10 +42,26 @@ function() use ($value){
                     return resolve(TypeRegistry::class)->get($type->name->value);
                 })->filter()->toArray();
             },
+            'resolveType' => $this->getResolver(function ($value, Context $context, ResolveInfo $info) {
```

Comment:
Resolvers have 4 arguments

Reply:
Alright, can you add a line comment to our code please?

Reply:
Sure!